### PR TITLE
Fix term crash bug, no auto-close finished term anymore

### PR DESCRIPTION
**Pull request by G-Rowell**
Hi all,
[fix for a bug I introduced](https://github.com/siduck76/NvChad/pull/167#issuecomment-883833117)

**Bug fix**
- Fix closing terminal with `<SHIFT> + x` that is the only buffer (and opened with `<CNTL> + t + t`
      Now closes properly
      Now terminals that are quit from within using `<CNTL> + d` no longer automatically close that term buffer, may fix later
- Also fixes the status line issue too

@siduck76 planning on having a wander through docs & posts regarding the terminal,
so feel free to pull this but I may work more

**tl;dr:** monkey made bug, monkey a8 bug

### CODE REVIEW

**Comment by siduck76** [comment]
Btw why removed laststatus?  It's needed to hide statusline in terminal buffers

**Comment by G-Rowell** [comment]
I thought it was good to have the statusline in all buffers?
And thought this was the issue you were referring to before?

Happy to modify

**Comment by G-Rowell** [comment]
Can you explain this a little please?
https://github.com/siduck76/NvChad/pull/167#issuecomment-883493434

What part of the screenshot is buggy?

**Comment by siduck76** [comment]
> Can you explain this a little please?
> [#167 (comment)](https://github.com/siduck76/NvChad/pull/167#issuecomment-883493434)
> 
> What part of the screenshot is buggy?

image

the statusline gets back to default ( neovim's statusline)

**Comment by G-Rowell** [comment]
Ahhhhh, now I see what you mean, will look into it
Ty

**Comment by siduck76** [comment]
> I thought it was good to have the statusline in all buffers?
> And thought this was the issue you were referring to before?
> 
> Happy to modify

yes but some buffers like nvimtree  and terminal dont need statusline at all ;)

**Review by siduck76 — APPROVED** [review]

**Review by siduck76 — APPROVED** [review]
